### PR TITLE
#264 add placeholder on search bar

### LINKED ISSUE #264: Add Placeholder text in Search Bar
Suggestion for placeholder text in Searchbar, for instance in stocktakes. Should tell the user what field they are sorting by i.e. "Search ITEM NAME".

### PR DESCRIPTION
**Pull request by Rebecca367**
Fixes #264

## Change summary

add text to searchbar easy for user to search items.

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] click on customer invoice page with search bar and filter item name/item code to see if it is working.
-  [ ] click on customer invoices page with search bar and filter item name/item code to see if it is working.
-  [ ] click on customer requisition page with search bar and filter item name/item code to see if it is working.
-  [ ] click on customer requisitions page with search bar and filter item name/item code to see if it is working.
- [ ] click on supplier invoice page with search bar and filter item name/item code to see if it is working.
- [ ] click on supplier invoices page with search bar and filter item name/item code to see if it is working.
-  [ ] click on supplier requisition page with search bar and filter item name/item code to see if it is working.
-  [ ] click on supplier requisitions page with search bar and filter item name/item code to see if it is working.
-  [ ] click on stocktake page with search bar and filter item name/item code to see if it is working.
-  [ ] click on stocktakeEdit page page with search bar and filter item name/item code to see if it is working.
-  [ ] click on stocktakeManage page with search bar and filter item name/item code to see if it is working.

### Related areas to think about
 filtering terms might be confusing. It will be good if it is consistent like sorting item name or item code cross all pages.

### CODE REVIEW

**Comment by Chris-Petty** [comment]
@Rebecca367 Just a note to remember to make sure the base branch is correct when making the PR (I forget all the time too, but usually more obvious with like 10,000 changes).

I'll review later today.

File: src/pages/CustomerRequisitionPage.js [hunk 1]
```diff
@@ -130,7 +130,11 @@ export const CustomerRequisition = ({
       <View style={pageTopSectionContainer}>
         <View style={pageTopLeftSectionContainer}>
           <PageInfo columns={pageInfoColumns} isEditingDisabled={isFinalised} />
-          <SearchBar onChangeText={onFilterData} value={searchTerm} />
+          <SearchBar
+            onChangeText={onFilterData}
+            value={searchTerm}
+            placeholder={`${generalStrings.searchBar} ${tableStrings.customerRequisitions}`}
```
Comment (Chris-Petty) [inline src/pages/CustomerRequisitionPage.js]:
```suggestion
            placeholder={`${generalStrings.searchBar} ${tableStrings.customer}`}
```

File: src/pages/SupplierInvoicePage.js [hunk 2]
```diff
@@ -131,7 +131,11 @@ export const SupplierInvoice = ({
       <View style={pageTopSectionContainer}>
         <View style={pageTopLeftSectionContainer}>
           <PageInfo columns={pageInfoColumns} isEditingDisabled={isFinalised} />
-          <SearchBar onChangeText={onFilterData} value={searchTerm} />
+          <SearchBar
+            onChangeText={onFilterData}
+            value={searchTerm}
+            placeholder={`${generalStrings.searchBar} ${tableStrings.name}`}
```
Comment (Chris-Petty) [inline src/pages/SupplierInvoicePage.js]: Do the values for the column show for you?

image

**Comment by Rebecca367** [comment]
@Chris-Petty 
what should I do from here?
It shows that there are conflicts but i couldn't see any in the branch....

**Comment by Chris-Petty** [comment]
@Rebecca367 In github desktop app try merging develop into your own branch- I'll help if you'd like

**Comment by Rebecca367** [comment]
@Chris-Petty 
OK,sure!  Another learning opportunity..LOL

**Comment by Rebecca367** [comment]
@Chris-Petty 
Thanks Chris for resolving the conflicts :)

File: src/pages/StocktakeManagePage.js [hunk 3]
```diff
@@ -116,7 +116,11 @@ export const StocktakeManage = ({
     <DataTablePageView>
       <View style={pageTopSectionContainer}>
         <View style={pageTopLeftSectionContainer}>
-          <SearchBar onChangeText={onFilterData} value={searchTerm} />
+          <SearchBar
+            onChangeText={onFilterData}
+            value={searchTerm}
+            placeholder={`${generalStrings.search_by} ${tableStrings.item_name} ${tableStrings.item_code}`}
```
Comment (Chris-Petty) [inline src/pages/StocktakeManagePage.js]:
```suggestion
            placeholder={`${generalStrings.search_by} ${tableStrings.item_name}, ${tableStrings.item_code}`}
```

Comment (Rebecca367) [reply src/pages/StocktakeManagePage.js]: @Chris-Petty 
done.. `3a2ae9ce`

**Pull request merged**